src/api/types.ts

```diff
@@ -1,4 +1,4 @@
-import { LearnMoreLink, TaskMapping } from '../reporting-config';
+import { LearnMoreLink, TaskDetails } from '../reporting-config';
```

Comment:
You can ignore the changes in this file: I just change some typing names to help differentiate entities (like tasks and  features, etc) as they come back from the API vs in the normalized state.